float32 roll
float32 pitch
float32 yaw
float32 gx
float32 gy
float32 gz
float32 ax
float32 ay
float32 az
float32 mx
float32 my
float32 mz

float32 frequence_capteur